includes/controls/groups/background.php

```diff
@@ -258,6 +258,25 @@ public function init_fields() {
 			],
 		];
 
+		$dimentions_options = [];
+		foreach ( get_intermediate_image_sizes() as $size ) {
+			$dimentions_options[ $size ] = $size;
+		}
+		$dimentions_options['full'] = 'full';
+
+		$fields['dimension'] = [
+			'label' => esc_html_x( 'Dimension', 'Background Control', 'elementor' ),
+			'type' => Controls_Manager::SELECT,
+			'options' => $dimentions_options,
```

Comment:
Translation?

Reply:
Not relevant anymore, the logic has changed.